compute_endpoint/tests/unit/test_worker.py

```diff
@@ -25,13 +25,14 @@ def large_result(size):
     return bytearray(size)
 
 
-def ez_pack_function(serializer, func, args, kwargs):
-    serialized_func = serializer.serialize(func)
-    serialized_args = serializer.serialize(args)
-    serialized_kwargs = serializer.serialize(kwargs)
-    return serializer.pack_buffers(
-        [serialized_func, serialized_args, serialized_kwargs]
-    )
+def sleeper(t: float):
```

Comment:
This function is odd, I know it was moved. Do we know why this is doing a spin loop?

Reply:
Yes; because the test function that utilizes it, `test_app_timeout` is testing the `execute_task` function in isolation.  That is, `execute_task` is not being run on a separate host or separate process, but is run within the pytest process.  The internal implementation that utilizes `GC_TASK_TIMEOUT` is simply a separate thread, but due to the GIL, the thread needs to come out of C-land to process the injected exception.

Reply:
Neat, that makes sense.